classfile.go

```diff
@@ -82,6 +82,12 @@ func (p *App) Static__2(pattern string, fs http.FileSystem, allowRedirect ...boo
 	p.StaticHttp(pattern, fs, allowRedirect...)
 }
 
+// custom delimiter,
+// example:{{  }} => ${  }$
+func (p *App) SetDelims(left, right string) {
```

Comment:
Why we should support custom delimiter?

Reply:
It conflicts with yap
vue:
```vue
<script setup>
import { ref } from 'vue'
const message = ref('Hello World!')
</script>
<template>
  <h1>{{ message }}</h1>
</template>
```

> to prevent conflicts with the front-end framework, developers can decide to change delimiter of the template
example vue:
https://github.com/goplus/community/issues/63#issuecomment-1918111850